android-components/components/browser/menu/src/main/res/values/strings.xml

```diff
@@ -12,5 +12,9 @@
     <!-- Label for add-ons sub menu item for add-ons manager -->
     <string name="mozac_browser_menu_addons_manager">Add-ons Manager</string>
     <!-- Content description for the action bar "up" button -->
-    <string name="action_bar_up_description">Navigate up</string>
+    <string name="action_bar_up_description" moz:removedIn="124" tools:ignore="UnusedResources">Navigate up</string>
+    <!-- Content description for the action bar "back" button -->
+    <string name="action_bar_back_description">Navigate back</string>
```

Comment:
nit: should we use the prefix `mozac_browser_menu`?

Reply:
I'd rather not touch that for now